src/main/java/ch/njol/skript/expressions/ExprItemThrower.java

```diff
@@ -0,0 +1,79 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.UUID;
+
+@Name("Item Thrower")
+@Description("The player that thrown the dropped item.")
+@Examples({
+	"broadcast the item thrower of all dropped items"
+})
+@Since("INSERT VERSION")
+public class ExprItemThrower extends SimplePropertyExpression<Entity, OfflinePlayer> {
+
+	static {
+		registerDefault(ExprItemThrower.class, OfflinePlayer.class, "[item] thrower", "entities");
+	}
```

Comment:
I don't know how I feel about `thrower` as a stand alone, since ThrowableProjectile (splash potion, snowball, enderpearl, ect..) seems like it could be used in same place as well